examples/otel-collector-sample-configurations/README.md

```diff
@@ -0,0 +1,25 @@
+# Purpose of this directory
+
+This folder provides sample [custom monitoring configurations](http://cloud/anthos-config-management/docs/how-to/monitor-config-sync-custom)
+for Config Sync. These examples are intended for your convenience. While Config
+Sync strives to keep them updated, always consult the [latest configuration](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/metrics/otel.go).
+
+# Available Configurations
+
+_otel-collector-monarch.yaml_: Serves as a template for exporting metrics
```

Comment:
suggest bold instead of underline. underline usually implies a link.

Reply:
Or even inline code maybe?

Reply:
This format is italic, not underline.

Reply:
Which is the usual format when quoting a book name or doc name? I assume.